src/planscape/boundary/management/commands/upload_boundary.py

```diff
@@ -12,44 +12,68 @@
 
 
 class Command(BaseCommand):
-    help = ('Uploads a boundary layer specified in the boundary configuration file '
-            'into the Planscape database.')
+    help = (
+        "Uploads a boundary layer specified in the boundary configuration file "
+        "into the Planscape database."
+    )
 
     def add_arguments(self, parser):
-        parser.add_argument('data_directory', nargs='?', type=str,
-                            help='Location of the boundary data.')
-        parser.add_argument('--boundary', nargs='?', type=str,
-                            help=('Name of the boundary to upload; if missing, all '
-                                  'boundaries are uploaded.'))
-        parser.add_argument('--configuration_file', nargs='?', type=str,
-                            help='Path to the boundary.json file; default is BASE_DIR/config/boundary.json.')
-        parser.add_argument('--force',
-                            type=bool, default=False, action=argparse.BooleanOptionalAction,
-                            help='Overwrite the boundary layer if it exists.')
-        parser.add_argument('--verbose',
-                            type=bool, default=True, action=argparse.BooleanOptionalAction,
-                            help='Show the commands to be run but do not run them.')
-        parser.add_argument('--strict',
-                            type=bool, default=True, action=argparse.BooleanOptionalAction,
-                            help='If false, ignore errors if a polygon cannot be converted and uploaded.')
+        parser.add_argument(
+            "data_directory", nargs="?", type=str, help="Location of the boundary data."
+        )
+        parser.add_argument(
+            "--boundary",
+            nargs="?",
+            type=str,
+            help=(
+                "Name of the boundary to upload; if missing, all "
+                "boundaries are uploaded."
+            ),
+        )
+        parser.add_argument(
+            "--configuration_file",
+            nargs="?",
+            type=str,
+            help="Path to the boundary.json file; default is BASE_DIR/config/boundary.json.",
+        )
+        parser.add_argument(
+            "--force",
+            type=bool,
+            default=False,
+            action=argparse.BooleanOptionalAction,
+            help="Overwrite the boundary layer if it exists.",
+        )
+        parser.add_argument(
+            "--verbose",
```

Comment:
I've always found this weird: --verbose typically means more output, without changing the run itself.  If you're comfortable with this, are you ok changing this to --dry_run ?

Reply:
This is weird for sure. So misleading that I even used while running. I will change it.

Reply:
Checking this again, it is actually `verbose`. The argument is used as input to LayerMapping. A Django utility to import GIS data.

You can check the docs here: https://docs.djangoproject.com/en/4.2/ref/contrib/gis/layermapping/#save-keyword-arguments

I'll update the description, which is incorrect.